include/picongpu/particles/atomicPhysics/localHelperFields/RejectionProbabilityCache.hpp

```diff
@@ -45,29 +45,49 @@ namespace picongpu::particles::atomicPhysics::localHelperFields
             pmacc::DataSpace<picongpu::simDim> superCellIdx) const
             -> std::enable_if_t<std::is_same_v<alpaka::Dev<T_Acc>, alpaka::DevCpu>>
         {
-            constexpr uint16_t numBins = T_RejectionProbabilityCache::numberBins;
+            constexpr uint32_t numBins = T_RejectionProbabilityCache::numberBins;
+            constexpr uint32_t numCells = T_RejectionProbabilityCache::numberCells;
 
             // check if overSubscribed
             bool overSubscription = false;
-            for(uint16_t i = 0u; i < numBins; i++)
+            for(uint32_t i = 0u; i < numBins; i++)
             {
-                if(rejectionProbabilityCache.rejectionProbability(i) > 0._X)
+                if(rejectionProbabilityCache.getRejectionProbabilityBin(i) > 0._X)
+                    overSubscription = true;
+            }
+            for(uint32_t i = 0u; i < numCells; i++)
+            {
+                if(rejectionProbabilityCache.getRejectionProbabilityCell(i) > 0._X)
                     overSubscription = true;
             }
 
             // print content
             std::cout << "rejectionProbabilityCache " << superCellIdx.toString(",", "[]");
             std::cout << " oversubcribed: " << ((overSubscription) ? "true" : "false") << std::endl;
-            for(uint16_t i = 0u; i < numBins; i++)
+            std::cout << "Bins:" << std::endl;
+            for(uint32_t i = 0u; i < numBins; i++)
```

Comment:
because the first one is checking if their is at least one over subscribed bin or cell, while the second one write the actual bin content to console. Admittedly I could have done both of them in one loop but I wanted the head of the histogram output to state whether the histogram was over subscribed and only below list the content of the bin. And since this code is only useful for debugging while executing serially, performance wasn't a priority for me.